libcxx/include/__config

```diff
@@ -1077,6 +1077,17 @@ __sanitizer_verify_double_ended_contiguous_container(const void*, const void*, c
 #    define _LIBCPP_HAS_NO_RTTI
 #  endif
 
+// Same as _LIBCXXABI_DTOR_FUNC in libcxxabi, we need this when working with ABI in exceptions code
+#  if defined(_WIN32)
+#    define _LIBCXX_DTOR_FUNC __thiscall
+#  else
+#    define _LIBCXX_DTOR_FUNC
+#  endif
+
+#  if !defined(_LIBCPP_HAS_NO_EXCEPTIONS) && !defined(_LIBCPP_HAS_NO_RTTI) && !defined(__APPLE__)
```

Comment:
Why is this disabled on Apple?

Reply:
I changed this `ifdef` to be an availability macros instead: its' enabled when building and disabled for Apple vendoring (because the functionality is not yet released)